README.md

```diff
@@ -31,7 +31,8 @@ To install the Scorecards GitHub Action, you need to:
 
 ### Authentication
 1. [Create a Personal Access Token](https://github.com/settings/tokens/new) with the following read permissions:
-    - Note: `Read-only token for OSSF Scorecard Action - myorg/myrepo` (Note: replace `myorg/myrepo` with the names of your organization and repository so you can keep track of your tokens.)
+    - Note 1: `Read-only token for OSSF Scorecard Action - myorg/myrepo` (Note: replace `myorg/myrepo` with the names of your organization and repository so you can keep track of your tokens.)
+    - Note 2: If you install Scorecard on a repository owned by an organization that uses [SAML SSO](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/about-authentication-with-saml-single-sign-on), be sure to [enable SSO](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on) for your PAT token.
```

Comment:
Consider moving the notes such that they are not part of the bulleted list. Right now, the bulleted points is a list of permissions and having `Note` in there might be confusing.

Reply:
Agreed. Let's put this below the screenshot (before "2. Copy the token."

Reply:
made some updates, ptal. The first note was actually note a note, it was the note fields for the token.

Reply:
LGTM!

Reply:
Yaay!